openwisp_firmware_upgrader/api/views.py

```diff
@@ -33,8 +33,9 @@ class OrgAPIMixin(ProtectedAPIMixin):
     def get_queryset(self):
         queryset = self.queryset.all().order_by('-pk')
```

Comment:
that `.order_by('-pk')` does not make a lot of sense (because we use UUIDs which are not sequential so the order would be random). Moreover, we also specified the default ordering recently for the endpoints. I think it can be removed.